samples/modules/lvgl/touch_draw/prj.conf

```diff
@@ -0,0 +1,16 @@
+CONFIG_LV_Z_MEM_POOL_SIZE=32768
+CONFIG_LV_Z_SHELL=y
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_DISPLAY=y
+CONFIG_DISPLAY_LOG_LEVEL_ERR=y
+
```

Comment:
Note: for the sample to work under the native plattforms `CONFIG_INPUT=y` is needed - maybe also add an overlay for it?